autoload/gist.vim

```diff
@@ -640,6 +640,29 @@ function! s:update_GistID(id) abort
   return ret
 endfunction
 
+" AnonymousPost function:
+"   Post new pastebin. This is a fallback because Gist no longer supports
+"   anonymous gists.
+"
+function! s:AnonymousPost(content) abort
+  let filename = s:get_current_filename(1)
+  let ext = expand('%:e')
+  redraw | echon 'Posting anonymously to ix.io... (Gist no longer supports anonymous posts)'
+  let data = {'f:1': a:content, 'name:1': filename, 'ext:1': ext}
+  let res = webapi#http#post('http://ix.io', data)
+  if res.status =~# '^2'
+    let loc = res.content
+    let loc = substitute(loc, '\n*$', '', 'g')
+    let loc = loc.'/'
+    redraw | echomsg 'Done: '.loc
+  else
+    let loc = ''
+    echohl ErrorMsg | echomsg 'Post failed: '. res | echohl None
+  endif
+  return loc
+endfunction
```

Comment:
Thanks. But I want to drop anonymous post.

Reply:
Ah alright then. It was my first time writing VimL 👍

Reply:
If you want to keep this, please do

```vim
if exists('g:gist_anonymous_fallback')
  return call('g:gist_anonymous_fallback', [a:content])
endif
" raise error?
```